web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -140,10 +143,14 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
   const [canSave, setCanSave] = useState(false);
   const [confirmOpen, setConfirmOpen] = useState(false);
   const [loading, setLoading] = useState(false);
+
   const [reviewStatus, setReviewStatus] = useState(ReviewStatus.CLOSED);
   const [periodToAdd, setPeriodToAdd] = useState({
     name: '',
-    reviewStatus: ReviewStatus.OPEN
+    reviewStatus: ReviewStatus.OPEN,
+    launchDate: null,
```

Comment:
I bet if you removed lines 151-153, everything would work the same because these properties would just default to being undefined and that is probably treated the same as `null` by the `DatePicker` component.

Reply:
I try to avoid leaving anything to chance, if it's ok I'd prefer to stick with null since undefined can at times cause larger issues.